src/hotspot/share/opto/phaseX.cpp

```diff
@@ -891,9 +891,12 @@ void PhaseIterGVN::verify_step(Node* n) {
 }
 
 void PhaseIterGVN::trace_PhaseIterGVN(Node* n, Node* nn, const Type* oldtype) {
+  const Type* newtype = type_or_null(n);
+  if (nn != n || oldtype != newtype) {
+    C->print_method(PHASE_AFTER_ITER_GVN_STEP, 4, n);
```

Comment:
Should we keep this at level 4 and move the parser generated dumps to a new level 5? I often add dumps for these steps during IGVN but I rarely ever need the parser generated dumps.

Reply:
Thanks for the feedback Christian, your suggestion makes sense. I will address this in a few weeks when I am back from vacation.

Reply:
Sounds good, thanks Roberto and enjoy your vacation!

Reply:
Done! Please re-review.